site2/docs/functions-develop.md

```diff
@@ -129,6 +121,12 @@ public interface SerDe<T> {
     byte[] serialize(T input);
 }
 ```
+SerDe works in the following way in Java Functions.
+- If the input and output topics have schema, Pulsar Functions use schema for SerDe.
+- If the input and output topics do not exist, Pulsar Functions adopt the following rules to determine SerDe:
```

Comment:
If there is no input and output topics, what the schema is used for? Do you mean that the input and output topics do not specify the schema?

Reply:
In some scenarios, we do not have input or output topics. 
Maybe I should change "and" into "or".